app/views/report_mailer/tables_report.html.erb

```diff
@@ -2,6 +2,15 @@
 - @message: text to put in the beginning of the email
 - @reports: an array of EmailableReport
 %>
+<header class="usa-header">
+  <%= image_tag(
+        asset_url('email/logo.png'),
+        alt: APP_NAME,
+        width: 155,
+        height: 21,
+      ) %>
+</header>
+<hr class="height-05 margin-bottom-4 border-transparent">
```

Comment:
For the changes proposed in my previous comment, it required corresponding changes to apply the background color class here. I'm guessing there's no color in your merged code.